### PR TITLE
Add Express instructions

### PR DESCRIPTION
**Pull request by sprice**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,33 @@ http.HandleFunc("/.well-known/acme-challenge/", func(w http.ResponseWriter, r *h
...
+  for (var key in process.env) {
+    if (key.startsWith('ACME_TOKEN_')) {
+      var num = key.split('ACME_TOKEN_')[1]
+      if (acmeToken === process.env['ACME_TOKEN_' + num]) {
+        return res.send(process.env['ACME_KEY_' + num])
+      }
+    }
+  }
+
+  return res.status(401).send()
```
Comment (dmathieu) [inline README.md]: Why 401?

File: README.md [hunk 2]
```diff
@@ -158,6 +158,33 @@ http.HandleFunc("/.well-known/acme-challenge/", func(w http.ResponseWriter, r *h
...
+
+Define the following route in your app.
+
+```js
+app.get('/.well-known/acme-challenge/:acmeToken', function(req, res, next) {
+  var acmeToken = req.params.acmeToken
+
+  if (process.env.ACME_KEY && process.env.ACME_TOKEN) {
+    if (acmeToken === process.env.ACME_TOKEN) {
+      return res.send(process.env.ACME_KEY)
```
Comment (dmathieu) [inline README.md]: Could we get a single return path?

File: README.md (hunk 1 above)
Comment (sprice) [inline README.md]: I see this is the wrong usage of 401. I'll update it to 404.

Comment (dmathieu) [inline README.md]: Thank you :)

**Comment by dmathieu** [comment]
Thank you Shawn. This looks awesome.
One very last comment. Could you make this pass jslint?

**Comment by sprice** [comment]
Is eslint or jshint okay?

**Comment by dmathieu** [comment]
The simplest one of them?

**Comment by sprice** [comment]
This passes both http://jshint.com/ and http://eslint.org/demo/ (the eslint errors are only due to `app`, `next` and `process` not being defined. They are provided by Express/Node.

**Comment by dmathieu** [comment]
Awesome. Thank you, and sorry for all the comments with such a simple thing. 🌷

**Pull request merged**